.circleci/config.yml

```diff
@@ -217,6 +217,7 @@ jobs:
                 name: Build
                 command: |
                   bundle exec fastlane build_and_upload_beta_release skip_confirm:true create_gh_release:true
+                  bundle exec fastlane build_and_upload_jetpack_for_app_store
```

Comment:
This addition matches the rename below 👍 and goes to [a lane](https://github.com/wordpress-mobile/WordPress-iOS/blob/40b3ecd2c643c3bed319484d39618b501f876af4/fastlane/Jetpack-Fastfile#L75-L103) that does upload to the Store 👍 👍 

Nitpick. I found the name a bit confusing because "App Store" made me think "production" and it was surprising to see it matching the "Beta" from the previous line.